option_handler.rb

```diff
@@ -0,0 +1,18 @@
+class OptionHandler
+  def call_option(option)
+    case option
+    when 1
+      list_books
+    when 2
+      list_people
+    when 3
+      create_person
+    when 4
+      create_book
+    when 5
+      create_rental
+    when 6
+      list_rentals
+    end
+  end
+end
```

Comment:
- [ ] Please fix your functionalities, none of your menu options is not working anymore, making your code efficient, and readable doesn't mean breaking up your code. Rather you should maintain the working of the app while making it more efficient and it will help you to be a better developer. 

![image](https://github.com/noelfoka/library-school/assets/66476329/e81d892a-6311-4f1e-ac14-23007d852e96)